server.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const helmet = require('helmet');
+const cookieParser = require('cookie-parser');
+const server = express();
+const db = require('./db');
+const noteRoutes = require('./api/routes/noteRoutes');
+const userRoutes = require('./api/routes/userRoutes');
+
+const port = process.env.PORT || 3040;
+
+server.use(helmet());
+server.use(express.json());
```

Comment:
All good here, just a stylistic consideration - and some linter setups _**require**_ the "require" statements be made first, mostly for readability of the flow down the page, e.g.:
```js
// PACKAGE requirements
const express = require('express');
const helmet = require('helmet');
const cookieParser = require('cookie-parser');
// FILE requirements
const db = require('./db');
const noteRoutes = require('./api/routes/noteRoutes');
const userRoutes = require('./api/routes/userRoutes');

const port = process.env.PORT || 3040;

const server = express();
server.use(helmet());
server.use(express.json());
```

From there sometimes I arrange them alphabetically, or maybe on their importance/order of use. Just a thought for ya - this is otherwise just fine 👍 I did test this change on my side and it worked fine. One thing to consider with style guidelines, however, you may encounter situations when hoisting and relative placement of declarations may take precedence over style.